scripts/sct_run_batch.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+##############################################################################
+#
+# Wrapper to processing scripts, which loops across subjects. Data should be
+# organized according to the BIDS structure:
+# https://github.com/sct-pipeline/spine_generic#file-structure
+#
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Authors: Julien Cohen-Adad, Chris Hammill
+#
+# About the license: see the file LICENSE.TXT
+##############################################################################
+
+from __future__ import absolute_import
+
+import os
+import sys
+import argparse
+import multiprocessing
+import subprocess
+import re
+import time
+
+from spinalcordtoolbox.utils import Metavar, SmartFormatter
+
+parser = argparse.ArgumentParser(
+    description='Wrapper to processing scripts, which loops across subjects. Subjects '
+    'should be organized as folders within a single directory. We recommend '
+                'following the BIDS convention (https://bids.neuroimaging.io/). '
+                'The processing script (task) should accept a subject directory as its only argument. '
+                'Additional information is passed via environment variables and the arguments '
+                'passed via `--task-args`',
+    formatter_class=SmartFormatter,
+    prog=os.path.basename(__file__).strip('.py'))
+
+parser.add_argument("--jobs", type=int, default=1,
```

Comment:
So far the (unofficial) convention in SCT for long flags is to only use single dash. An increasing number of recent functions have used that convention so it would be great to do it for this function as well

Reply:
Happy to change over, although I do like the argparse auto short flags, so `-j` can be used instead of `--jobs`

Reply:
that sounds like a good discussion for our next SCT meeting ;-) (added to the agenda)

Reply:
Converted in my last commit.

Reply:
I don't feel strongly about single vs. double tick, but happy to talk about it at the next meeting